src/Elastic.Apm/Api/IExecutionSegment.cs

```diff
@@ -31,6 +31,8 @@ public interface IExecutionSegment
 
 		/// <summary>
 		/// A flat mapping of user-defined labels with string values.
+		/// Dots (<code>.</code>) in the keys are not allowed. In case you have a <code>.</code> in your label key, it will be replaced by <code>_</code>.
+		/// For example <code>foo.bar</code> will be stored as <code>foo_bar</code>.
```

Comment:
It might be worth clarifying the term `stored` since the key is still "stored" with dots in `Labels` `Dictionary`. Maybe `stored in Elasticsearch`?

Reply:
👍  - added it to the text.